src/Schema/Directives/Fields/PaginateDirective.php

```diff
@@ -138,6 +138,10 @@ protected function getPaginatedResults(array $resolveArgs, int $page, int $first
             $resolveArgs[3]
         );
 
+        if ($query instanceof \Laravel\Scout\Builder) {
```

Comment:
Maybe import `\Laravel\Scout\Builder`?

Reply:
I think that in some scenarios, when we check for an ```instanceof``` common class names like ```Builder```, it seems more readable to put the full namespace. 

```php
if ($query instanceof \Laravel\Scout\Builder) {
    return $query->paginate($first, 'page', $page);
}

if ($query instanceof Builder) {
    return $query->paginate($first, 'page', $page);
}
```

Open for discussion...

Reply:
We always import classes in code. In this case aliasing to `ScoutBuilder` should make things more readable.